src/main/java/com/squareup/javapoet/JavaFile.java

```diff
@@ -218,6 +230,25 @@ public Builder addFileComment(String format, Object... args) {
       return this;
     }
 
+    public Builder addStaticImport(Enum<?> constant) {
```

Comment:
I'm getting closer and closer... it's been [$E](https://github.com/sormuras/javapoet/commit/ba36bff2656b7215d440239b0c73065a3a0cfdfd), it's still [$R](https://github.com/square/javapoet/pull/388) ... for now, I can live with **$T**, when it comes to static imports.